testsuite/MDAnalysisTests/utils/test_nuclinfo.py

```diff
@@ -32,66 +32,62 @@
-                                  err_msg="RNA hydroxyl dihedrals do not match")
-
-    def test_pseudo_dihe_baseflip(self):
-        seg1 = self.universe.residues[3].atoms.segids[0]
-        seg2 = self.universe.residues[19].atoms.segids[0]
-
-        # There is not really a baseflip, just testing the code...
-        flip = nuclinfo.pseudo_dihe_baseflip(self.universe, 4, 20, 5, seg1=seg1, seg2=seg2, seg3=seg1)
-        assert_almost_equal(flip, 322.0826, self.prec,
-                            err_msg="pseudo_dihedral for resid 5 against 4--20 do not match")
+from numpy.testing import assert_almost_equal ,assert_array_almost_equal
+
+
+@staticmethod    
+@pytest.fixture()
+def universe():
+    return MDAnalysis.Universe(NUCL)
+
+def test_wc_pair(universe):
+    seg1 = universe.residues[3].atoms.segids[0]
+    seg2 = universe.residues[19].atoms.segids[0]
+    wc = nuclinfo.wc_pair(universe, 4, 20, seg1, seg2)
+    assert_almost_equal(wc, 2.9810174, err_msg="Watson-Crick distance does not match expected value.")
+
+def test_major_pair(universe):
+    seg1 = universe.residues[3].atoms.segids[0]
+    seg2 = universe.residues[19].atoms.segids[0]
+    maj = nuclinfo.major_pair(universe, 4, 20, seg1, seg2)
+    assert_almost_equal(maj, 2.9400151, err_msg="Watson-Crick distance does not match expected value.")
+
+def test_minor_pair(universe):
+    seg1 = universe.residues[3].atoms.segids[0]
+    seg2 = universe.residues[19].atoms.segids[0]
+
+    minor = nuclinfo.minor_pair(universe, 4, 20, seg1, seg2)
+    assert_almost_equal(minor, 3.7739358, err_msg="Watson-Crick distance does not match expected value.")
+
+def test_torsions(universe):
+    nucl_acid = np.array(nuclinfo.tors(universe, "RNAA", 4), dtype=np.float32)
+    expected_nucl_acid = np.array(
+        [296.45596313, 177.79353333, 48.67910767, 81.81109619, 205.58882141, 286.37353516, 198.09187317],
+        dtype=np.float32)
+    assert_almost_equal(nucl_acid, expected_nucl_acid, 4,
+                        err_msg="Backbone torsion does not have expected values for "
+                                "alpha, beta, gamma, epsilon, zeta, chi.")
+
+def test_hydroxyl(universe):
+    hydroxyls = np.array([nuclinfo.hydroxyl(universe,
+                                               universe.atoms.segids[0], resid)
+                             for resid in (7, 10, 11, 22)])
+    expected_hydroxyls = np.array(
+        [ 122.73991394,  123.34986115,  123.20658112,  122.57156372],
+        dtype=np.float32)
+    assert_array_almost_equal(hydroxyls, expected_hydroxyls, 4,
+                              err_msg="RNA hydroxyl dihedrals do not match")
+
+def test_pseudo_dihe_baseflip(universe):
+    seg1 = universe.residues[3].atoms.segids[0]
+    seg2 = universe.residues[19].atoms.segids[0]
+
+    # There is not really a baseflip, just testing the code...
+    flip = nuclinfo.pseudo_dihe_baseflip(universe, 4, 20, 5, seg1=seg1, seg2=seg2, seg3=seg1)
+    assert_almost_equal(flip, 322.0826, 4,
```

Comment:
We could do a keyword argument. I'm not generally in favour of adding variables to the global namespace.